packages/blade/src/components/List/ListItemText.tsx

```diff
@@ -0,0 +1,17 @@
+import type { ReactElement } from 'react';
+import type { TextProps, TextVariant } from '../Typography';
+import { Text } from '../Typography';
+import { useListContext } from './ListContext';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+
+type ListItemTextProps = Omit<TextProps<{ variant: TextVariant }>, 'variant' | 'size'>;
+
+const _ListItemText = ({ children, testID, ...props }: ListItemTextProps): ReactElement => {
+  const { size } = useListContext();
+
+  return <Text {...props} size={size} children={children} testID={testID} />;
```

Comment:
we can just do

```
<Text {...props} size={size} />;
```

do we need to destructure and set again? 🤔

Reply:
It should be fine since Text destructures it anyway